v8/plantuml/README.md

```diff
@@ -0,0 +1,30 @@
+This plugin converts [PlantUML](https://plantuml.com/) files.
+
+The default configuration will output all `*.puml` files found under the `pages` dir as SVG files.
+
+Developed against PlantUML version 1.2020.24.  Probably works with some earlier versions.
+
+# Unicode
+
+The plugin expects PlantUML files to be encoded with UTF-8.
+
+# Known Issues
+
+- It's slow!  Every PlantUML rendering launches a new Java process, on my laptop it takes 4-8 seconds per file.
+  I have some ideas to speed this up, and they may be available in future plugin versions.
```

Comment:
What are those ideas? Mine would be a PlantUML compile server that starts up with the first PlantUML file encountered, and that is reused for the rest — but note that this might be tricky with parallel builds (`nikola build -n 2`). (You are free to implement whatever you want.)

Reply:
I have it working with [JPype](https://jpype.readthedocs.io/en/latest/) in a separate branch but I'd like to add it later as a separate PR, this one feels big enough already.

Was amazingly simple to do because JPype just worked for me.  From the Python perspective, rendering PlantUML is a simple function call into the Java.

It wont be the right solution for everyone so I'm planning a config option something like `PLANTUML_RUNNER` which can be set to `exec` or `jpype` to select how PlantUML is run.

The compile server was my second option but is more fiddly to code - shutting down the process / tcp connections etc.